### ISSUE TITLE
[Proposal] Support global readonly account

### ISSUE THREAD

**Issue by annProg** [goharbor/harbor#9209]
**Is your feature request related to a problem? Please describe.**
When using a private image repository in Kubernetes, I want to use a read-only account for security reasons. It can be done based on the robot account or project members, but it is cumbersome to add members for each project.

**Describe the solution you'd like**
I hope to have a global read-only account that can pull all the images.

And when using non-database authentication, it should continue to use database authentication like the `admin` account.

**Describe the main design/architecture of your solution**

**Describe the development plan you've considered**

**Additional context**

**Comment by xaleeks** [comment]
@annProg can you ping me to discuss this? there are a few others similar to your requirement, ie a global robot.  my email: xalex@vmware.com or ping me xalex on cncf slack

**Comment by xaleeks** [comment]
I'm temporarily tagging this as 2.0 if we have room to tackle this but not commited

**Comment by lalithvaka** [comment]
Any update on this feature? Definitely be useful for inline scanning of external image scanners.

**Comment by xaleeks** [comment]
@lalithvaka  it's going to be covered here goharbor/community#148

**Comment by lalithvaka** [comment]
Thank you @xaleeks . I am currently using the latest v2.0.2 in our Prod instance with Sysdig harbor inline scanner. I have created a robot account under one of the projects and configured the [sysdig inline scanner](https://github.com/sysdiglabs/harbor-scanner-sysdig-secure) and seems to work for all the other projects too. same robot account is able to pull all the images under different projects and scan. I was expecting a global robot readonly account to get this work!!!. 
cc @nestorsalceda

**Comment by xaleeks** [comment]
@lalithvaka  great to hear this!!

this progress will be tracked in https://github.com/goharbor/harbor/issues/10774
